examples/app.py

```diff
@@ -48,17 +48,13 @@
 
 # Create Flask application
```

Comment:
can you add a minimum documentation to quick start as e.g. in oauthclient?

https://github.com/inveniosoftware/invenio-oauthclient/blob/master/examples/app.py#L28-L42